R/kernel_Factor.R

```diff
@@ -202,7 +202,7 @@ FactorKernel <- R6::R6Class(
       y <- y[self$xindex]
       stopifnot(x>=1, y>=1, x<=self$nlevels, y<=self$nlevels,
                 abs(x-as.integer(x)) < 1e-8, abs(y-as.integer(y)) < 1e-8)
-      if (x==y) {
```

Comment:
This is an extra issue that I ran into. There is one other changed here but some more that I didn't touch (with `xx == yy`) because I didn't encounter triggers for them